extensions/amp-next-page/amp-next-page.md

```diff
@@ -89,9 +89,42 @@ component will render a maximum of three documents (total) on screen at one sing
 
 [tip type="important"]
 **Important** [`<amp-analytics>`](../amp-analytics/amp-analytics.md) is [currently unsupported](https://github.com/ampproject/amphtml/issues/15807) on pages users land on through `<amp-next-page>`.
-Measuring pageviews is supported through [`<amp-pixel>`](../../builtins/amp-pixel.md).
+Measuring pageviews is supported through [`<amp-pixel>`](../../builtins/amp-pixel.md) or `<amp-analytics>` on the host page.
 [/tip]
 
+### Analytics triggers
+
+Partial support for analytics is included through the initial host page through two events that `<amp-next-page>` triggers and that you can track in your [amp-analytics](https://amp.dev/documentation/components/amp-analytics) config:
+
+| Event                  | Fired when                                               |
+| ---------------------- | -------------------------------------------------------- |
+| `amp-next-page-scroll` | The user scrolls to a new page                           |
+| `amp-next-page-click`  | The user click on an article from the recommendation box |
+
+Both of the `triggers` come with two variables `fromUrl` and `toUrl` that refer to the previous and the current page that the user is viewing, which can be used as follows:
```

Comment:
nit: wording, `previous` and `current` in context don't require specifying the user

```md
Both triggers provide the variables `fromUrl` and `toUrl`, which refer to the URLs of
the previous and current pages. They can be used as follows:
```